### PR TITLE
feat(notifications): add multi email recipients

### PR DESCRIPTION
**Pull request by teschiopol**
If I need to translate all label, please let me know and I'll try my best. 

If something is not on your coding style, I'll fix it. 

Ty

### CODE REVIEW

**Comment by teschiopol** [comment]
I mispell the branch, it's issue 436

**Comment by ellite** [comment]
Thanks.
I've been sick and then quite busy trying to catch up at work. I'll try to look at everything that happened on GitHub this week. I've seen quite a few emails.
I'll review this as soon as possible.

File: endpoints/cronjobs/sendcancellationnotifications.php [hunk 1]
```diff
@@ -42,6 +42,7 @@
         $email['smtpUsername'] = $row["smtp_username"];
         $email['smtpPassword'] = $row["smtp_password"];
         $email['fromEmail'] = $row["from_email"] ? $row["from_email"] : "wallos@wallosapp.com";
+        $mail['otherEmail'] = $row["other_email"];
```
Comment (ellite) [inline endpoints/cronjobs/sendcancellationnotifications.php:45]: `$email `vs $mail

File: endpoints/cronjobs/sendnotifications.php [hunk 2]
```diff
@@ -60,6 +60,7 @@
         $email['smtpUsername'] = $row["smtp_username"];
         $email['smtpPassword'] = $row["smtp_password"];
         $email['fromEmail'] = $row["from_email"] ? $row["from_email"] : "wallos@wallosapp.com";
+        $mail['otherEmail'] = $row["other_email"];
```
Comment (ellite) [inline endpoints/cronjobs/sendnotifications.php:63]: `$email` vs $mail

**Review by ellite — CHANGES_REQUESTED** [review]
Thanks. 
Just saw two small mistakes, if you don't mind fixing them.
I will handle the remaining translations. Thanks.

**Comment by teschiopol** [comment]
Done and sorry.

**Comment by ellite** [comment]
Nothing to be sorry about.
Thank you very much.

When I have some time I will test and merge.

Regards.

**Review by ellite — APPROVED** [review]
Thank you.

**Pull request merged**